tests/test_abstract_inst/test_optical_spectrum_analyzer.py

```diff
@@ -78,15 +72,25 @@ def test_osa_start_sweep(osa):
 # OSAChannel #
 
 
-def test_osa_channel_wavelength(osc):
+@pytest.mark.parametrize("num_ch", [1, 5])
+def test_osa_channel_wavelength(osa, num_ch):
     """Channel wavelength method: ensure existence."""
-    inst = osc()
-    with pytest.raises(NotImplementedError):
-        inst.wavelength()
+    with expected_protocol(osa, [], []) as inst:
+        inst._channel_count = num_ch
+        ch = inst.channel[0]
+        with pytest.raises(NotImplementedError):
```

Comment:
call single/mutli channel instrument through the inst.channel ProxyList function and ensure proper forwarding